features/support/hooks.js

```diff
@@ -37,33 +36,31 @@ module.exports = function () {
     // application server mock
     this.app = makeExpressApp(this)
     this._server = http.createServer(this.app)
-    var primusOpts = {
+    const primusOpts = {
```

Comment:
`const` should probably not be defined inside a method. Pull this to the top or change it to var/let